docs/compatibility/sdks.md

```diff
@@ -34,67 +34,53 @@ The following table gives an overview of compatibility and support of EDOT SDKs
     <tbody>
```

Comment:
The table is broken with this change, need to adapt the rowspan / colspan in the `<tr>` above